src/main.js

```diff
@@ -58,13 +58,29 @@ function showFormPage() {
   homeButton.classList.remove('hidden');
   saveCoverButton.classList.add('hidden');
   randomCoverButton.classList.add('hidden');
+  savedView.classList.add('hidden');
 };
 function showSavedCovers() {
+  // will display the hardcoded example, not pulling in freshly saved array items
+  savedView.classList.remove('hidden');
+  hiddenForm.classList.add('hidden');
   savedCoversPage.classList.remove('hidden');
   coverView.classList.add('hidden');
   randomCoverButton.classList.add('hidden');
   saveCoverButton.classList.add('hidden');
   homeButton.classList.remove('hidden');
+  savedCoversSec.innerHTML = "";
```

Comment:
Nice use of interpolation! Can bring the closing section tag tab spacing back one to the same indentation as the opening.